.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+### how to use
+Leave the headings unless they dont apply to your PR, replace the explanatory text with text describing your PR.
+
+#### Summary
+A one-line description of your change that will be extracted and added to the [project changelog](../data/changelog.txt).  
+The format is ```SUMMARY: Category "description"```  
+Category is one of Features, Content, Interface, Mods, Balance, Bugfixes, Performance, Infrastructure, Build, I18N
```

Comment:
Is this case-sensitive?

Reply:
It is currently, but it doesn't have to be.